web/controllers/processes/wait.ex

```diff
@@ -7,9 +7,12 @@ defmodule Dwylbot.WaitProcess do
 
   def delay(error, payload, event_type, token) do
     Process.sleep(error.wait)
-    data = @github_api.get_data(token, payload, event_type)
-    check_errors = Rules.check_errors(data, event_type)
-    Rules.any_error?(check_errors, error)
-    && @github_api.report_error(token, error)
+    if error.verify do
```

Comment:
This condition allow a rule to comment (change label, change assignees,...) directly on Github without having to check if the error is still here after a period of time. For the merge conflict rule we want dwylbot to comment directly as we don't need to check after 30s if the PRs still have merge conflicts.